@LinProp/LinProp.m

```diff
@@ -273,17 +273,17 @@ function display(obj)
             else
                 if obj.IsComplex
                     sreal = ['(' num2str(abs(get_value(real(obj))), df) ...
-                             ' � ' num2str(get_stdunc(real(obj)), df) ')'];       
+                             ' ± ' num2str(get_stdunc(real(obj)), df) ')'];       
                     simag = ['(' num2str(abs(get_value(imag(obj))), df) ...
-                             ' � ' num2str(get_stdunc(imag(obj)), df) ')'];
+                             ' ± ' num2str(get_stdunc(imag(obj)), df) ')'];
                     if (get_value(imag(obj)) < 0)
                         s = [sreal ' - ' simag 'i'];
                     else
                         s = [sreal ' + ' simag 'i'];
                     end
                 else        
                     s = ['(' num2str(abs(get_value(obj)), df) ...
-                         ' � ' num2str(get_stdunc(obj), df) ')'];
+                         ' ± ' num2str(get_stdunc(obj), df) ')'];
```

Comment:
LinProp.m was an ANSI file and you changed it to a UTF-8 file. That's why GitHub shows here a different. Please change it back to ANSI.

Reply:
I commited the file saved again as ANSI. Is that sufficient or should I try to revert the erronous commit that changed the file to UTF-8? (I am not sure if/how that is possible.)

Reply:
That's ok for me.